cogs/annual_handover_and_reset.py

```diff
@@ -0,0 +1,225 @@
+"""Contains cog classes for annual handover and role reset functionality."""
+
+
+from collections.abc import Sequence
+
+__all__: Sequence[str] = ("HandoverCommandCog", "AnnualResetRolesCommandCog")
+
+import logging
+from logging import Logger
+from typing import Final
+
+import discord
+
+from db.core.models import GroupMadeMember
+from utils import AllChannelTypes, CommandChecks, TeXBotApplicationContext, TeXBotBaseCog
+
+logger: Logger = logging.getLogger("TeX-Bot")
+
+
+class HandoverCommandCog(TeXBotBaseCog):
+    """Cog class that defines the handover command."""
```

Comment:
Use updated command name, speech marks & forward-slash: `...the "/committee-handover" command."""`